{"cert_user_id": "saber28@github", "next_comment_id": 3, "comment": {"1604064736_mirrored_HelloZeroNet_github": [{"comment_id": 1, "body": ">Storage\r\n\r\nOne big file\r\n> Uploading? WebSocket suitable for this or separate HTTP post request?\r\n\r\nHTTP post request\r\n>Compressed piecemap?\r\n\r\nMsgpack", "added": 1502234275, "modified": 1502234299, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/7#issuecomment-321133405", "source_type": "github"}], "1604067608_mirrored_ZeroNetTickBot_github": [{"comment_id": 2, "body": "GPLv3 and Lax", "added": 1575336884, "modified": 1575336884, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/2273#issuecomment-560994243", "source_type": "github"}]}, "next_topic_id": 7, "topic": [{"topic_id": 1604065828, "title": "Python win64 OpenSSL error", "body": "[14:18:49] - OpenSSL load failed: [Error 126] , falling back to slow bitcoin verify\r\n[14:18:49] - Patching sockets to tor socks proxy: 127.0.0.1:9050\r\n[14:18:49] - Version: 0.5.3 r2003, Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)], Gevent: 1.2.1\r\n\r\n>pip list\r\nDEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.\r\nasn1crypto (0.22.0)\r\ncffi (1.10.0)\r\ncryptography (1.8.1)\r\nenum34 (1.1.6)\r\ngevent (1.2.1)\r\ngreenlet (0.4.12)\r\nidna (2.5)\r\nipaddress (1.0.18)\r\nmsgpack-python (0.4.8)\r\npackaging (16.8)\r\npip (9.0.1)\r\npycparser (2.17)\r\npyOpenSSL (16.2.0)\r\npyparsing (2.2.0)\r\npyzmq (16.0.2)\r\nsetuptools (28.8.0)\r\nsix (1.10.0)\r\n\r\nPython 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)] on win32\r\nType \"help\", \"copyright\", \"credits\" or \"license\" for more information.\r\n>>> import ssl\r\n>>> print ssl.OPENSSL_VERSION\r\nOpenSSL 1.0.2j  26 Sep 2016\r\n\r\ndebug.log\r\n[2017-03-25 19:25:21,131] DEBUG    - Disable SSL compression failed: [Error 126]  (normal on Windows)\r\n[2017-03-25 19:25:21,131] DEBUG    - Missing SSLwrap, readded.\r\n[2017-03-25 19:25:21,131] DEBUG    - Python SSL version: OpenSSL 1.0.2j  26 Sep 2016", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1490412175, "modified": 1502874782, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/872", "source_type": "github"}, {"topic_id": 1604065928, "title": "Port always shows open, even if it's closed", "body": "Because port checker site switched to https", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1495594146, "modified": 1495603838, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/947", "source_type": "github"}, {"topic_id": 1604065930, "title": "Client downloads and distributes files from muted spammers", "body": "Can we stop client distributing files from muted spammers?", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1495594323, "modified": 1504700181, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/948", "source_type": "github"}, {"topic_id": 1604066167, "title": "Unable to start after update to newest version", "body": "- Starting ZeroNet...\r\n[06:55:05] - OpenSSL loaded, version: 0100020CF\r\n[06:55:05] - Unhandled exception: argument --autodownload_bigfile_size_limit: conflicting option string(s): --autodownload_bigfile_size_limit\r\nTraceback (most recent call last):\r\n  File \"zeronet.py\", line 18, in main\r\n    import main\r\n  File \"/root/zeronet/src/main.py\", line 108, in <module>\r\n    config.loadPlugins()\r\n  File \"/root/zeronet/src/Config.py\", line 379, in loadPlugins\r\n    ConfigPlugin(self)\r\n  File \"/root/zeronet/src/Config.py\", line 374, in __init__\r\n    self.createArguments()\r\n  File \"plugins/Zeroname/UiRequestPlugin.py\", line 30, in createArguments\r\n    return super(ConfigPlugin, self).createArguments()\r\n  File \"plugins/UiPassword/UiPasswordPlugin.py\", line 115, in createArguments\r\n    return super(ConfigPlugin, self).createArguments()\r\n  File \"plugins/OptionalManager/OptionalManagerPlugin.py\", line 136, in createArguments\r\n    return super(ConfigPlugin, self).createArguments()\r\n  File \"plugins/Bigfile/BigfilePlugin.py\", line 680, in createArguments\r\n    return super(ConfigPlugin, self).createArguments()\r\n  File \"plugins/BigFile/BigfilePlugin.py\", line 678, in createArguments\r\n    group.add_argument('--autodownload_bigfile_size_limit', help='Also download bigfiles until this limit if help distribute option is checked', default=1, metavar=\"MB\", type=int)\r\n  File \"/usr/lib/python2.7/argparse.py\", line 1308, in add_argument\r\n    return self._add_action(action)\r\n  File \"/usr/lib/python2.7/argparse.py\", line 1509, in _add_action\r\n    action = super(_ArgumentGroup, self)._add_action(action)\r\n  File \"/usr/lib/python2.7/argparse.py\", line 1322, in _add_action\r\n    self._check_conflict(action)\r\n  File \"/usr/lib/python2.7/argparse.py\", line 1460, in _check_conflict\r\n    conflict_handler(action, confl_optionals)\r\n  File \"/usr/lib/python2.7/argparse.py\", line 1467, in _handle_conflict_error\r\n    raise ArgumentError(action, message % conflict_string)\r\nArgumentError: argument --autodownload_bigfile_size_limit: conflicting option string(s): --autodownload_bigfile_size_limit\r\n", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1507622373, "modified": 1507630335, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1139", "source_type": "github"}, {"topic_id": 1604066392, "title": "Sites with UTF title unable to open", "body": "### Step 1: Please describe your environment\r\n\r\n  * ZeroNet version: 3350\r\n  * Operating system: ZeroNet client run on Debian 9, browser run on windows 10 and android 6\r\n  * Web browser: Chrome 64.0.3282.186\r\n  * Tor status: enabled\r\n  * Opened port: yes\r\n  * Special configuration: UI Password, Caddy reverse proxy\r\n\r\n### Step 2: Describe the problem:\r\n\r\n#### Steps to reproduce:\r\n\r\n  1. Update to 3350\r\n  2. Visit sites with UTF characters in title, like nekox.bit\r\n\r\n#### Observed Results:\r\n\r\n  * What happened? This could be a screenshot, a description, log output (you can send log/debug.log file to hello@zeronet.io if necessary), etc.\r\n\r\nUnable to load the site\r\n\r\n#### Possible Reasons:\r\n\r\n`window.self !== window.top` may work improperly when cross-OS and UTF characters in title, \r\n\r\n#### Expected Results:\r\n\r\n  * What did you expect to happen?\r\nOpen sites with UTF characters in title like versions before 3350", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1520392327, "modified": 1520403867, "source_link": "https://github.com/HelloZeroNet/ZeroNet/issues/1315", "source_type": "github"}, {"topic_id": 1604066521, "title": "Improve Orbot Compatibility", "body": "Use password auth first to work with orbot", "parent_topic_uri": "1604062980_users_1Cy3ntkN2GN9MH6EaW6eHpi4YoRS2nK5Di", "added": 1525474731, "modified": 1526970268, "source_link": "https://github.com/HelloZeroNet/ZeroNet/pull/1413", "source_type": "github"}]}